DocSum/docker_image_build/build.yaml

```diff
@@ -49,15 +49,8 @@ services:
       dockerfile: comps/llms/src/doc-summarization/Dockerfile
     extends: docsum
     image: ${REGISTRY:-opea}/llm-docsum:${TAG:-latest}
-  vllm:
```

Comment:
Please do not delete `vllm` and `vllm-gaudi` images, that's used for xeon and gaudi test.